vizro-core/examples/default/app.py

```diff
@@ -496,6 +500,7 @@ def create_home_page():
     """Function returns the homepage."""
     page_home = vm.Page(
         title="Homepage",
+        description="Welcome to the Vizro demo app",
```

Comment:
This is just a placeholder.  What would you like the app description to be on the  homepage?

Reply:
Maybe this? wdyt @huong-li-nguyen? For context, this is what will appear in social media preview when someone shares the link to vizro.mckinsey.com.

```suggestion
        description="Vizro demo app for studying gapminder data",
```

Reply:
Each site has a slightly different format.  Here's a preview from metatags.io. 

Some (like LinkedIn) don't show the description at all.  The image and the title are the most important items.  The real link would be `https://vizro.mckinsey.com/`  I think it would also look nice to include a smaller "by McKinsey" in the image if you wanted to promote the company more.


For example, here's how it would look on Twitter:
![image](https://github.com/mckinsey/vizro/assets/72614349/13124eae-c520-4bb0-8326-ebd478c1afe2)

And on Slack:
![image](https://github.com/mckinsey/vizro/assets/72614349/af0e4c42-3bfd-4b86-a77d-59216e838974)

Reply:
Looks great to me - thanks you two!! 🚀

Reply:
Agree this could be nice with a "by McKinsey" or "by QuantumBlack Labs" or something, but that will take quite a while to do since it would need to go through branding, discussions about whether it's McKinsey or QB, figuring out the right design, etc., and everyone is on holiday now... So I'm very happy to leave the image as we have it 🙂